extensions/saga/runtime/pom.xml

```diff
@@ -32,6 +32,7 @@
 
     <properties>
         <camel.quarkus.jvmSince>1.1.0</camel.quarkus.jvmSince>
+        <camel.quarkus.nativeSince>1.2.0</camel.quarkus.nativeSince>
```

Comment:
Should be native since 1.4.0.

Reply:
Thanks @jamesnetherton and I just fix it.